app/config.py

```diff
@@ -179,6 +179,7 @@ class Config(object):
     MOU_SIGNED_ON_BEHALF_SIGNER_RECEIPT_TEMPLATE_ID = 'c20206d5-bf03-4002-9a90-37d5032d9e84'
     MOU_SIGNED_ON_BEHALF_ON_BEHALF_RECEIPT_TEMPLATE_ID = '522b6657-5ca5-4368-a294-6b527703bd0b'
     MOU_NOTIFY_TEAM_ALERT_TEMPLATE_ID = 'd0e66c4c-0c50-43f0-94f5-f85b613202d4'
+    NOTIFY_NUMBER_SMS_SENDER = '07984404008'
```

Comment:
small detail, can we call this var something like `NOTIFY_INTERNATIONAL_SMS_SENDER` so it's obvious what it's used for?

Reply:
good idea - sometimes naming is hard